FluentTc.Tests/AcceptanceTests.cs

```diff
@@ -1163,5 +1163,131 @@ public void GetSingleSuccessfulFinishedBuild()
             // Assert
             builds.Single().Id.Should().Be(123);
         }
+
+        [Test]
+        [TestCase("A simple description")]
+        [TestCase("")]
+        public void SetProjectDescription(string description)
+        {
+            // Arrange
+            var teamCityCaller = CreateTeamCityCaller();
+
+            var connectedTc = new RemoteTc().Connect(_ => _.AsGuest(), teamCityCaller);
+
+            // Act
+            connectedTc.SetProjectConfigurationField(_ => _.Id("ProjectId"), __ => __.Description(description));
+
+            // Assert
+            A.CallTo(
+                () =>
+                    teamCityCaller.Put(description,
+                    HttpContentTypes.TextPlain, "/app/rest/projects/id:ProjectId/description", string.Empty))
+                        .MustHaveHappened(Repeated.Exactly.Once);
+        }
+
+        [Test]
+        [TestCase("A simple description")]
+        [TestCase("")]
+        public void SetBuildDescription(string description)
+        {
+            // Arrange
+            var teamCityCaller = CreateTeamCityCaller();
+
+            var connectedTc = new RemoteTc().Connect(_ => _.AsGuest(), teamCityCaller);
+
+            // Act
+            connectedTc.SetBuildConfigurationField(_ => _.Id("BuildId"), __ => __.Description(description));
+
+            // Assert
+            A.CallTo(
+                () =>
+                    teamCityCaller.Put(description,
+                    HttpContentTypes.TextPlain, "/app/rest/buildTypes/id:BuildId/description", string.Empty))
+                        .MustHaveHappened(Repeated.Exactly.Once);
+        }
+
+        [Test]
+        [TestCase(true)]
+        [TestCase(false)]
+        public void SetProjectArchived(bool archived)
+        {
+            // Arrange
+            var teamCityCaller = CreateTeamCityCaller();
+
+            var connectedTc = new RemoteTc().Connect(_ => _.AsGuest(), teamCityCaller);
+
+            // Act
+            connectedTc.SetProjectConfigurationField(_ => _.Id("ProjectId"), __ => __.Archived(archived));
+
+            // Assert
+            A.CallTo(
+                () =>
+                    teamCityCaller.Put(archived.ToString(System.Globalization.CultureInfo.InvariantCulture).ToLower(),
+                    HttpContentTypes.TextPlain, "/app/rest/projects/id:ProjectId/archived", string.Empty))
+                        .MustHaveHappened(Repeated.Exactly.Once);
+        }
+
+        [Test]
+        [TestCase(true)]
+        [TestCase(false)]
+        public void SetBuildDescription(bool paused)
+        {
+            // Arrange
+            var teamCityCaller = CreateTeamCityCaller();
+
+            var connectedTc = new RemoteTc().Connect(_ => _.AsGuest(), teamCityCaller);
+
+            // Act
+            connectedTc.SetBuildConfigurationField(_ => _.Id("BuildId"), __ => __.Paused(paused));
```

Comment:
In terms of more readable API I would prefer.

```C# 
connectedTc.SetBuildConfigurationField(_ => _.Id("BuildId"), __ => __.Paused());
```
and 
```C#
connectedTc.SetBuildConfigurationField(_ => _.Id("BuildId"), __ => __.NotPaused());
```

The same for Archived

Thanks!